cg/store/models.py

```diff
@@ -1075,6 +1075,37 @@ class IlluminaSequencingRun(InstrumentRun):
     __mapper_args__ = {"polymorphic_identity": DeviceType.ILLUMINA}
 
 
+class PacBioSequencingRun(InstrumentRun):
+    __tablename__ = "pacbio_sequencing_run"
+
+    id: Mapped[int] = mapped_column(ForeignKey("instrument_run.id"), primary_key=True)
+    well: Mapped[Str32 | None]
```

Comment:
is the well on the plate?

Reply:
The smart cell has a plate-well id that is like `1_A01`. The first number is the plate, the `A01` is the well